bloom/README

```diff
@@ -1,17 +1,24 @@
-Scripts:
---------
+## Scripts:
+
+### Create a new gazeboX_rosY_pkgs repository
+
+For a new official wrappers the notation used below correspond to:
+`gazeboX_rosY_pkgs` (`X` is major version in Gazebo, `Y` is 1 or 2 for ROS)
+
+ 1. Create new repo under github.com/osrf/ organization named:
+    - https://github.com/osrf/gazeboX_rosY_pkgs-release
```

Comment:
I don't see any `ros1` repos. Should it say that `Y` is empty for ROS 1?

Reply:
098fc2c